example/lib/develop/peripheral_list/components/rssi_view.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:flutter/material.dart';
+
+class RssiView extends StatelessWidget {
+  final String rssiValue;
+
+  RssiView({this.rssiValue});
+
+  @override
+  Widget build(BuildContext context) {
```

Comment:
actually it is a single text, is it sensible to create a standalone widget from that?